src/components/hmi_message_handler/src/messagebroker_adapter.cc

```diff
@@ -161,8 +164,8 @@ void MessageBrokerAdapter::ProcessRecievedFromMB(Json::Value& root) {
     return;
   }
 
-  Json::FastWriter writer;
-  std::string message_string = writer.write(root);
+  Json::StreamWriterBuilder writer;
+  std::string message_string = Json::writeString(writer, root);
```

Comment:
@ydementieiev const?

Reply:
@AByzhynar done in [1345b2e](https://github.com/smartdevicelink/sdl_core/pull/3155/commits/1345b2e4f52415e896a5fa3d232aa391c69751bb)